components/hab/src/main.rs

```diff
@@ -51,10 +51,13 @@ use habitat_core::{crypto::{init,
                    env::{self as henv,
                          Config as _},
                    fs::{cache_artifact_path,
+                        cache_key_path,
+                        cache_path,
```

Comment:
Is there a commit missing? I don't see any definition of `cache_path` in https://github.com/habitat-sh/habitat/blob/master/components/core/src/fs.rs.

Reply:
Ah yes, these are coming from the PR that this PR is dependent upon to merge first.
https://github.com/habitat-sh/habitat/pull/6945

When it merges I planned to rebase on master.

Reply:
In the meantime, can you rebase on top of `ma/hab_6878/pkg_download`? That will allow CI to run and simplify the review process.

Reply:
Of course, gladly! Thanks for your patience with me. Rebase is now complete 😄